### PR TITLE
Large scale cel

### PR DESCRIPTION
**Pull request by Jean-KOUAGOU**
- Added python scripts to run TDL and Drill on DBpedia learning problems with `"https://dbpedia.data.dice-research.org/sparql"` as knowledge source. 

- Updated README with `Bash` commands to start class expression learning

- Also fixed some typos in README, e.g., `Learining OWL Class Expressions` instead of  `Learining OWL Class Expression`

- I can add the `Bash` command for DL-Learner as well before merge, but execution freezes after some minutes. It is unable to solve even a single learning problem

### CODE REVIEW

File: examples/dbpedia_concept_learning_with_drill_and_triplestore.py
Comment (Demirrr) [inline examples/dbpedia_concept_learning_with_drill_and_triplestore.py:1]: I don't think that we need have two different script that only differs by selected learner.
Perhaps, you can use argparse to select learner

File: examples/large_scale_learning_problem_generation.py
Comment (Demirrr) [inline examples/large_scale_learning_problem_generation.py:1]: I don't think that there is a need to merge a script into devl branch that doesn't work.
So, either ensure that this code is compartiable with the dev branch or remove it from your PR.

File: ontolearn/binders.py
Comment (Demirrr) [inline ontolearn/binders.py:1]: Your changes in the __init__ decrease the quality of code by 
1. adding more lines than required
2. multiple assertions without unique information

File: ontolearn/binders.py [hunk 1]
```diff
@@ -348,3 +351,316 @@ def fit_from_iterable(self, dataset: List = None, max_runtime=None) -> List[Dict
             self.max_runtime = max_runtime
 
         return [self.fit(pos=p, neg=n, max_runtime=self.max_runtime).best_hypothesis() for (s, p, n) in dataset]
+    
+    
+class DLLearnerBinderSPARQL:
```
Comment (Demirrr) [inline ontolearn/binders.py:356]: We do not need to create yet another binder. 
Please ensure that the content of this class in DLLearnerBinder.

**Review by Demirrr — CHANGES_REQUESTED** [review]

File: examples/dbpedia_concept_learning_with_drill_and_triplestore.py
Comment (Jean-KOUAGOU) [reply examples/dbpedia_concept_learning_with_drill_and_triplestore.py:1]: > I don't think that we need have two different script that only differs by selected learner. Perhaps, you can use argparse to select learner

I merged both files into `dbpedia_concept_learning_with_ontolearn.py`

File: ontolearn/binders.py (hunk 1 above)
Comment (Jean-KOUAGOU) [reply ontolearn/binders.py]: > We do not need to create yet another binder. Please ensure that the content of this class in DLLearnerBinder.

Remove DLLearnerBinderSPARQL, added its content into DLLearnerBinder

File: README.md [hunk 2]
```diff
@@ -118,6 +118,20 @@ print(owl_expression_to_sparql(expression=h))
 save_owl_class_expressions(expressions=h,path="owl_prediction")
 ```
 
+- With one command
```
Comment (Demirrr) [inline README.md:121]: Please remove these lines. There is no need to repeat exact same commands multiple times

File: ontolearn/binders.py
Comment (Jean-KOUAGOU) [reply ontolearn/binders.py:1]: > Your changes in the **init** decrease the quality of code by
> 
> 1. adding more lines than required
> 2. multiple assertions without unique information

Resolved, see below

File: README.md (hunk 2 above)
Comment (Jean-KOUAGOU) [reply README.md:121]: Should I remove all three commands?

Comment (Jean-KOUAGOU) [reply README.md]: Done. I removed the three commands

**Pull request closed without merging**